docker/k8s-first-steps/server_run.sh

```diff
@@ -22,7 +22,7 @@ echo ">>> Setting up the db for Django" >> /app/server_run.log
 python manage.py migrate >> /app/server_run.log
 
 echo ">>> Starting frontend build process in the background" >> /app/server_run.log
-cd frontend && yarn start &
+cd frontend && npm start &
```

Comment:
This looks surprising. Not in the sense that it's new, but that it's not symmetric to `tag-admin`

Reply:
Good point. I'll need to review this script in a bit more detail.